### ISSUE TITLE
Make constant values for use with the default image parameter

### ISSUE THREAD

**Issue by bsoyka** [bsoyka/gravify#24]
Take a look at @charanjit-singh's comment [here](https://github.com/bsoyka/gravify/issues/3#issuecomment-537601348).

There are 8 different constants that need to be added in some way, and you can see them [here](https://en.gravatar.com/site/implement/images#default-image).

Please comment if you'd like to work on this, and make a pull request to the `development` branch.

**Comment by mfaheemakhtar** [comment]
Hey @bsoyka, I can take it.

What do you think about using 
```
DEFAULT_IMAGE = {
  '404': '404',
  'MYSTERY_PERSON': 'mp',
  'IDENTICON': 'identicon',
  'MONSTERID': 'monsterid',
  'WAVATAR': 'wavatar',
  'RETRO': 'retro',
  'ROBOHASH': 'robohash',
  'BLANK': 'blank'
}
```

instead of 

```
  Type_404 = "404"
  Type_mystreyPerson = "mp"
```

**Comment by bsoyka** [comment]
Hey, @mfaheemakhtar! I like your idea! A couple small changes, let me know your thoughts:
```python
DEFAULT_IMAGES = {
  '404': '404',
  'mystery_person': 'mp',
  'identicon': 'identicon',
  'monsterid': 'monsterid',
  'wavatar': 'wavatar',
  'retro': 'retro',
  'robohash': 'robohash',
  'blank': 'blank'
}
```
Also, when you make your PR, make sure you add a test or two to make sure it works.

**Comment by mfaheemakhtar** [comment]
Hey @bsoyka , how do you run unit tests?

`python3 ./tests/test_main.py`

**Comment by bsoyka** [comment]
You can run the tests with this:
```
python -m unittest tests/test_main.py
```

**Comment by bsoyka** [comment]
Closed via #25
